demisto_sdk/commands/doc_reviewer/tests/doc_reviewer_test.py

```diff
@@ -1,82 +1,553 @@
+from os import path
 from typing import List
 
+import pytest
+
+from demisto_sdk.commands.common.constants import FileType
+from demisto_sdk.commands.common.tools import find_type
 from demisto_sdk.commands.doc_reviewer.doc_reviewer import DocReviewer
 from TestSuite.json_based import JSONBased
 
 
-def test_doc_review_with_release_notes_is_skipped_on_invalid_yml_file(malformed_integration_yml):
+class TestDocReviewFilesAreFound:
+    """
+    Tests scenarios in which files are found before performing doc-review.
     """
-    Given -
-        malformed yml integration file.
 
-    When -
-        Calling doc-review with --release-notes.
+    def test_find_files_from_invalid_path(self):
```

Comment:
Even though you are right, I think it could be a bit of a problem to parametrize it because each time I need a different value from the fixture that is being passed to the test. I also think it would be good to separate the tests names logically as it would make more sense to the person reading it.